src/app/bungie-api/authenticated-fetch.ts

```diff
@@ -118,7 +123,7 @@ export async function getActiveToken(): Promise<Tokens> {
   try {
     token = await (cache || getAccessTokenFromRefreshToken(token.refreshToken!));
     setToken(token);
-    console.log('Successfully updated auth token from refresh token.');
+    infoLog('Successfully updated auth token from refresh token.');
```

Comment:
I wonder if the signature should require at least 1 string, otherwise the entire message is a 'tag'

`infoLog(tag: string, ...args: any[])` -> `infoLog(tag: string, message: string, ...args: any[])`

Reply:
probably overkill, but would it be worth moving the tags to an enum in the util ?